src/components/RichTextEditor/EditorPositioningWrapper/PlateWrapper.tsx

```diff
@@ -1,17 +1,23 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import React, { forwardRef } from 'react';
-import { WrapperProps } from './types';
+import { Position, WrapperProps } from './types';
 import { EditorPositioningWrapper } from '.';
 import { useRichTextEditorContext } from '../context/RichTextEditorContext';
+import { merge } from '@utilities/merge';
 
 export const PlateWrapper = forwardRef<HTMLDivElement, WrapperProps>(({ children }, ref) => {
-    const { position } = useRichTextEditorContext();
+    const { position, border } = useRichTextEditorContext();
 
-    const classNames = EditorPositioningWrapper[position].PlateWrapperClassNames;
```

Comment:
My suggestion, put the logic into the context and exchange position with classNames so that context return you only classNames instead of position and border and you get this independent.

Reply:
fixed